modin/pandas/test/utils.py

```diff
@@ -203,6 +204,16 @@
 test_data_categorical_values = list(test_data_categorical.values())
 test_data_categorical_keys = list(test_data_categorical.keys())
 
+# Fully fill all of the partitions used in tests.
+test_data_large_categorical_dataframe = {
+    i: pandas.Categorical(np.arange(NPartitions.get() * 32))
+    for i in range(NPartitions.get() * 32)
```

Comment:
We should have a variable for minimal partition size.

Reply:
I created https://github.com/modin-project/modin/pull/3751 for that.

Reply:
@mvashishtha , thanks!